### ISSUE TITLE
Roll Me - The drag & drop interrupts sometimes

### ISSUE THREAD

**Issue by desertking** [ruffle-rs/ruffle#11613]
### Describe the bug

The game seems to work fine but the drag & drop function to rotate the worlds sometimes stutters and freezes at least on my pc making it very hard to play. errors thrown:

Loading SWF file https://www.spiele-umsonst.de/azad/downloads/flashgames/rollme.swf

ERROR core/src/avm2/events.rs:419 Error dispatching event EventObject(EventObject { type: "enterFrame", class: flash.events::Event, ptr: 0xa7bffc }) to handler FunctionObject(FunctionObject { ptr: 0x27dada4 }) : TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: gotoAndPlay)

Also some text error?

ERROR core/src/display_object/movie_clip.rs:2126 Got "ReferenceError: Error #1069: Property flashx.textLayout.container::TextContainerManager not found on global and there is no default value." when constructing AVM2 side of movie clip of type fl.text::TCMText

### Expected behavior

To rotate fluently? 

### Affected platform

Online demo

### Operating system

Win 10, PC

### Browser

Newest Chrome

**Comment by desertking** [comment]
You are right it works really bad even with Flashplayer. That's strange. Can't think of I ever would have added the game if it was working that bad during the time. Dunno if something broke with a new flash version or if it broke on something else.

**Comment by ChrisCPI** [comment]
Looks like this might have regressed at some point? It appeared to be fine in a 2023 build, but now in the latest version the logo intro is constantly looping.
